### ISSUE TITLE
Feature Request -  Ability to brand "MeshMessenger" with company name and logo

### ISSUE THREAD

**Issue by LPJon** [Ylianst/MeshCentral#2199]
I would like see branding in this area so that my customers know they are speaking with my company. Something like "MeshMessenger - \<logo image\> - \<Company Name\>" would work. I realize you may want to keep your branding as well which is why I left "MeshMessenger" in there. However, if you are willing you could remove that.

**Comment by Ylianst** [comment]
Made this change in GitHub, it will be available in MeshCentral v0.7.52. You can customize the MeshMessenger title bar with these two new options in the domain section of the config.json:

```
"meshMessengerTitle": "MeshMessenger",
"meshMessengerPicture": "messenger.png"
```

The image should be 38px in height and placed in the "meshcentral-data" folder. Let me know if that works.

![MC2-MessengerCustomization](https://user-images.githubusercontent.com/1319013/105677975-6ad4ab00-5ea1-11eb-9f6c-b19e76b5f7c1.png)

**Comment by LPJon** [comment]
@Ylianst I can confirm this is working. Thanks!
